includes/class-wc-swedbank-plugin.php

```diff
@@ -0,0 +1,554 @@
+<?php
+
+namespace SwedbankPay\Payments\WooCommerce;
+
+defined( 'ABSPATH' ) || exit;
+
+use SwedbankPay\Payments\WooCommerce\WC_Swedbank_Pay_Transactions;
+use SwedbankPay\Payments\WooCommerce\WC_Background_Swedbank_Pay_Queue;
+use WC_Order;
+use WC_Admin_Meta_Boxes;
+use Exception;
+
+class WC_Swedbank_Plugin {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class WC_Swedbank_Plugin has 21 non-getter- and setter-methods. Consider refactoring WC_Swedbank_Plugin to keep number of methods under 10.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)